exercises/rail-fence-cipher/rail_fence_cipher_test.exs

```diff
@@ -0,0 +1,91 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("rail_fence_cipher.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule RailFenceCipherTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "test to encode with two rails" do
+    assert RailFenceCipher.encode("XOXOXOXOXOXOXOXOXO", 2) == "XXXXXXXXXOOOOOOOOO"
+  end
+
+  @tag :pending
+  test "test to encode with three rails" do
```

Comment:
All of the test descriptions begin with `test` which makes the test read "test test to encode with three rails". Remove `test` from the descriptions and perhaps consider rewording. (test "encoding with three rails"?)